src/engine/sys/sdl_glimp.cpp

```diff
@@ -1882,6 +1885,62 @@ static void GLimp_InitExtensions()
 	// made required in OpenGL 3.0
 	glConfig2.textureFloatAvailable = LOAD_EXTENSION_WITH_TEST( ExtFlag_CORE, ARB_texture_float, r_ext_texture_float.Get() );
 
+	glConfig2.internalFormatQuery2Available = LOAD_EXTENSION_WITH_TEST( ExtFlag_NONE, ARB_internalformat_query2, r_arb_internalformat_query2.Get() );
+
+	if ( glConfig2.internalFormatQuery2Available )
+	{
+		GLint64 param;
+		glGetInternalformati64v( GL_TEXTURE_2D, GL_RGBA16, GL_FRAMEBUFFER_BLEND, 1, &param );
+		switch( param )
+		{
+			case GL_TRUE:
+				glConfig2.textureRGBA16BlendAvailable = 1;
+				break;
+			case GL_FALSE:
+				glConfig2.textureRGBA16BlendAvailable = 0;
+				break;
+			default:
+				/* Mesa returning GL_FULL_SUPPORT looks to be a bug, it should return GL_TRUE, returning
```

Comment:
Maybe the change to use internalFormatQuery2 is not worth it if we have to work around another bug?

Reply:
Answered there about the worthness of it:

- https://github.com/DaemonEngine/Daemon/pull/1235#issuecomment-2286471659